src/core/hle/service/hid/controllers/gesture.cpp

```diff
@@ -1,19 +1,29 @@
-// Copyright 2018 yuzu emulator team
+// Copyright 2021 yuzu Emulator Project
 // Licensed under GPLv2 or any later version
 // Refer to the license.txt file included.
 
-#include <cstring>
-#include "common/common_types.h"
 #include "common/logging/log.h"
+#include "common/math_util.h"
 #include "common/settings.h"
 #include "core/core_timing.h"
 #include "core/frontend/emu_window.h"
 #include "core/hle/service/hid/controllers/gesture.h"
 
+namespace {
+constexpr inline f32 Square(s32 num) {
```

Comment:
```suggestion
constexpr f32 Square(s32 num) {
```

since this function already has internal linkage, and inline signifies external linkage.

This namespace should also be inside the Service::HID namespace.